### ISSUE TITLE
Unable to reproduce the experimental results in the paper

### ISSUE THREAD

**Issue by orange-ang** [noahzn/Lite-Mono#95]
Hello, thanks for great code.I got the pretrained model and I ran the train.py to reproduce the lite-mono, but I got different results.
paper
abs_rel | sq_rel | rmse | rmse_log | a1 | a2 | a3 |
0.107 0.765 4.561 0.183 0.886 0.963 0.983
own
abs_rel | sq_rel | rmse | rmse_log | a1 | a2 | a3 |
0.107 0.793 4.578 0.184 0.887 0.962 0.982
and best(?) in 20
 abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 |
0.106  &   0.777  &   4.579  &   0.183  &   0.887  &   0.963  &   0.983
Among them, the second and third indicators are a little different from those in the paper.
My train is on a single 2080ti,and the environments are as below.

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
_openmp_mutex             4.5                       2_gnu    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
blas                      1.0                         mkl    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates           2023.7.22            hbcca054_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
cudatoolkit               11.0.3              h7761cd4_12    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
cudnn                     8.4.0.27             hed8a83a_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
cycler                    0.11.0                   pypi_0    pypi
freetype                  2.12.1               h267a509_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
imageio                   2.31.2                   pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jpeg                      9b                   h024ee3a_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
kiwisolver                1.4.5                    pypi_0    pypi
libedit                   3.1.20191231         he28a2e2_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libffi                    3.2.1             he1b5a44_1007    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libgcc-ng                 13.2.0               h807b86a_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libgomp                   13.2.0               h807b86a_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libpng                    1.6.39               h753d276_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libstdcxx-ng              13.2.0               h7e041cc_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libtiff                   4.0.9                he6b73bb_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libuv                     1.46.0               hd590300_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
libzlib                   1.2.13               hd590300_5    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
linear-warmup-cosine-annealing-warm-restarts-weight-decay 1.0                      pypi_0    pypi
matplotlib                3.3.4                    pypi_0    pypi
mkl                       2021.4.0           h06a4308_640    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl-service               2.4.0            py37h402132d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
mkl_fft                   1.3.1            py37h3e078e5_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
mkl_random                1.2.2            py37h219a48f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
ncurses                   6.4                  h59595ed_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
networkx                  2.6.3                    pypi_0    pypi
ninja                     1.11.1               h924138e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
numpy                     1.21.6                   pypi_0    pypi
olefile                   0.46               pyh9f0ad1d_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
opencv-python             4.5.2.54                 pypi_0    pypi
openssl                   1.1.1w               hd590300_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
pillow                    8.3.2                    pypi_0    pypi
pip                       23.3.1             pyhd8ed1ab_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
protobuf                  3.19.0                   pypi_0    pypi
pyparsing                 3.1.1                    pypi_0    pypi
python                    3.7.4                h265db76_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.7                     2_cp37m    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
pytorch                   1.7.1           py3.7_cuda11.0.221_cudnn8.0.5_0    pytorch
pywavelets                1.3.0                    pypi_0    pypi
readline                  7.0               hf8c457e_1001    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
scikit-image              0.15.0                   pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
setuptools                68.2.2             pyhd8ed1ab_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
six                       1.15.0                   pypi_0    pypi
sqlite                    3.33.0               h62c20be_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tensorboardx              2.1                      pypi_0    pypi
thop                      0.1.1-2209072238          pypi_0    pypi
timm                      0.4.12                   pypi_0    pypi
tk                        8.6.13          noxft_h4845f30_101    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
torchaudio                0.7.2                      py37    pytorch
torchvision               0.8.2                py37_cu110    pytorch
typing_extensions         4.7.1              pyha770c72_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
vc                        14.2                 h21ff451_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vs2015_runtime            14.27.29016          h5e58377_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wheel                     0.41.3             pyhd8ed1ab_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
xz                        5.2.6                h166bdaf_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
zlib                      1.2.13               hd590300_5    https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge_
So what details do I need to pay attention to in order to reproduce the result of the lite-mono model？

**Comment by noahzn** [comment]
Hi, your results are already very close to or even better than those reported in the paper. As each batch of the training is different, you will not get exactly the same results as reported in the paper. Each time your training may converge on a different local minimum.

**Comment by orange-ang** [comment]
Have you ever got similar or worse sq_rel and rmse? As it is still a bit worse than the results on the paper, so I'm trying to get better results. And as you mentioned in your previous issue, the best results might be achieved at epoch 16 or 17 as the training can converge fast. But I noticed my results seems to not be like this.(the best results in epoch 20 has a _Scaling_ ratios | med: 33.036), a bit different from the later epochs., so I just want to confirm this and then may help me check the problem).
I'll try to increase drop_rate as you advise to see whether it can get better results. Anyway, thanks for your quick apply.

**Comment by noahzn** [comment]
Hi, yes, it happened when the network converged to a 'bad' local minimum. Sometimes it helps when you set the learning rate to, e.g., `[0.0001, 5e-6, 16, 0.0001, 1e-5, 16])`. This means that the learning rate will be reset at epoch 16, and it may make the training jump out of the local minimum.

**Comment by noahzn** [comment]
I'm closing the issue due to lack of response.
